outport/process/outportDataProvider.go

```diff
@@ -66,6 +69,8 @@ func NewOutportDataProvider(arg ArgOutportDataProvider) (*outportDataProvider, e
 	}, nil
 }
 
+var log = logger.GetOrCreate("outport/process/outportDataProvider")
```

Comment:
can you declare the logger at the beginning of the file

Reply:
done